Source/OxyPlot/Axes/AxisUtilities.cs

```diff
@@ -106,8 +106,18 @@ public static IList<double> FilterRedundantMinorTicks(IList<double> majorTicks,
             var ret = new List<double>();
             var previousMinorTick = 0d;
             var j = 1;
+
             var currentMajorTick = majorTicks.Count > 1 ? majorTicks[j] : majorTicks[0];
 
+            static double GetEpsilon(double tick1, double tick2)
+            {
+                return Math.Abs(tick1 - tick2) * 1e-3;
```

Comment:
I wonder if this could have issues with large values for `LogarithmicAxis.Base` (e.g. on the order of 10/1e-3).

Reply:
Interesting thought!
I don't really know of an application where such large bases would be used, but of course that doesn't mean at all that they don't exist.

Just for fun I modified the LogarithmicAxis>Tick Calculation example by multiplying the bases by 1000. The result is kind of 'interesting':
![image](https://user-images.githubusercontent.com/18097859/83324373-2129b700-a265-11ea-8584-284460909eab.png)

So yes I do think `AxisUtilities.FilterRedundantMinorTicks(...)` could have a problem with these kinds of bases, but I don't think we really support them anyway.

Reply:
That first image looks like a bug in computing the minor tick interval rather than your stuff: did you happen to see what happens with the old code?

I agree this shouldn't be a blocker either way.

Reply:
The first image is because the `LogarithmicAxis` falls back to linearly-spaced ticks when the range is too small. That's what I meant with we don't support these kinds of bases anyway.

Before this PR I reckon this would have looked pretty much the same.